src/lib/translator.ts

```diff
@@ -53,9 +49,14 @@ export class Translator implements TranslatorInterface {
         //   if none found find by short locale like "de"
         //   if still none found search by the fallback which is by default "en"
         let translated = "";
-        [this.long(), this.short(), this._options.fallback].find(
-            (tag) => !!pattern.find((pat) => !!(translated = this._resources[`${tag}.${pat}`])),
-        );
+        [this.long(), this.short(), this._options.fallback].some((tag) => {
+            const localPattern = [...pattern];
+            if (typeof options.count === "number") {
```

Comment:
i think this is always string because of line 48

Reply:
the type of options.count never changes ;)

Reply:
yes I misread it :D